content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -166,17 +166,25 @@ is restarted, the tracked image age is reset, causing the kubelet to wait the fu
 based on image age.
 {{< /note>}}
 
-### Container garbage collection {#container-image-garbage-collection}
+### Container garbage collection {#container-image-garbage-collection} (deprecated)
+
+{{< caution >}}
+Container garbage collection is deprecated in favour of `--eviction-hard` or
+`--eviction-soft` and will be removed in a future version. For more details on
+eviction policy, see [node-pressure-eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/).
+{{< /caution >}}
```

Comment:
These command line arguments are deprecated, and the format isn't right.
We also [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future), although we do make reasonable exceptions to that general guidance.

Try this:
```suggestion
### Legacy garbage collection for containers (deprecated) {#container-image-garbage-collection}

{{< feature-state for_k8s_version="v1.9" state="deprecated" >}}

{{< note >}}
Container garbage collection is deprecated, in favor of using the kubelet configuration file
entries `evictionHard` and `evictionSoft.
`--eviction-soft` and will be removed in a future version. For more details on
eviction policy, see [node-pressure-eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/).
{{< /note >}}
```

:information_source: I am *not* sure about which version deprecated setting these command line arguments.